src/common/Navigation/styled.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+export const NavigationWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1920px;
+  margin: 0 auto;
+  height: 94px;
+  background-color: #000;
+`;
+
+export const NavigationList = styled.ul`
+  display: inline-flex;
+  align-items: center;
+`;
+
+export const NavigationTitle = styled.li`
+  font-size: 24px;
+  font-weight: 500;
+  text-decoration: none;
+  color: #fff;
+`;
+
+export const NavigationItem = styled.li`
+  font-size: 14px;
+  font-weight: 600;
+  text-decoration: none;
+  color: #fff;
+`;
```

Comment:
Tutaj jest w zasadzie ten sam kod. Dla wygody i jakości kodu można go zapisać przekazując style z NavigationTitle do NavigationItem i zamienić to co trzeba. Mniej więcej tak export const NavigationItem = styled(NavigationTitle)`` i tyle.
Też potem colory trzeba będzie zapisać z pomocą theme, jeśli robiłeś git pulla to powienieneś to mieć w core/App